merkle-tree/src/deployer.js

```diff
@@ -4,6 +4,10 @@
 @author iAmMichaelConnor
 */
 
+/*
+THIS FILE LOOKS TO BE DEPRECATED AND CAN BE REMOVED. The merkle-tree microservice will not deploy a contract. It will only subscribe to already-deployed contracts.
```

Comment:
Any reason you don't want to remove the file now in this case?

Reply:
I have no logical reason. Good point. I'll remove it :)

Reply:
Change made